src/app/Http/Controllers/Operations/ListOperation.php

```diff
@@ -71,32 +71,44 @@ public function search()
     {
         $this->crud->hasAccessOrFail('list');
 
+        // how many total entries there is applying the controller/model constrains, before any filtering or search
+        $this->crud->setOperationSetting('totalEntryCount', $this->crud->getTotalEntryCount());
+
```

Comment:
Any reason to do this HERE, at the top of the function @pxpm and not lower, where we calculate the totals? I don't see it used until then anyway, and it's odd to read something like:
- calculate total entry count
- do something else
- do another thing
- yet another thing
- now let's continue calculating the total entry count

Reply:
Fixed in https://github.com/Laravel-Backpack/CRUD/pull/4695 - please review & merge if you think it's ok @pxpm

Reply:
I replied to this question in https://github.com/Laravel-Backpack/CRUD/pull/4695#discussion_r986689198